CHANGELOG.md

```diff
@@ -6,13 +6,9 @@
 - Version 22.4.x will be the last series to support Java 11. Version 22.7.0 will require Java 17 to build and run.
 - In the Besu EVM Library all references to SHA3 have been renamed to the more accurate name Kecack256, including class names and comment. [#3749](https://github.com/hyperledger/besu/pull/3749)
```

Comment:
Can you add the RC2 breaking change here too

In the Besu EVM Library all references to SHA3 have been renamed to the more accurate name Keccak256, including class names and comment. https://github.com/hyperledger/besu/pull/3749

Reply:
nvm